src/snyk/common/languageServer/settings.ts

```diff
@@ -61,6 +62,7 @@ export class LanguageServerSettings {
       activateSnykCodeQuality: `${codeQualityEnabled}`,
       activateSnykOpenSource: `${ossEnabled}`,
       activateSnykIac: `${iacEnabled}`,
+      enableDeltaFindings: `${configuration.getDeltaFindingsEnabled()}`,
```

Comment:
This needs a protocol version update in LS. So we first need to change LS protocol version, release it and then update VSCode, Eclipse & IntelliJ.